server/app.js

```diff
@@ -0,0 +1,44 @@
+const createError = require("http-errors");
+const express = require("express");
+const path = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+
+const controllers = require("./controllers");
+
+const app = express();
+
+app.use(logger("dev"));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.use(controllers);
+
+/**
+ * Uncomment these lines after setting up react server
+ * 
+ * app.use(express.static(path.join(__dirname,'..', 'client', 'build')));
+ * app.get('/', (req, res) => {
+ *  res.sendFile(path.join(__dirname,'..', 'client', 'build', 'index.html'));
+ * });
+ * 
+*/ 
+
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
+  next(createError(404));
+});
+
```

Comment:
I don't think this would be necessary as 404 is handled by react-router in the front and this will never get matched
@RamyAlshurafa @ashatat @FarahZaqout 
What do you think?

Reply:
what if someone sends a request through another service than the web?
like postman? what react-router would send to it?

Reply:
Idk, we should try and see how is goes